sentry_sdk/tracing.py

```diff
@@ -194,25 +189,45 @@ def __exit__(self, ty, value, tb):
         self.finish(hub)
         scope.span = old_span
 
-    def new_span(self, **kwargs):
+    def start_child(self, **kwargs):
```

Comment:
Isn't this a breaking change for manual instrumentation?
In JS we made sure it's not breaking even if old API is used.